packages/common-soroban/src/project/models.ts

```diff
@@ -14,12 +14,99 @@ import {
   SubqlCustomDatasource,
   FileReference,
   CustomDataSourceAsset,
-  SubqlEventHandler,
+  SorobanBlockFilter,
+  SorobanTransactionFilter,
+  SorobanOperationFilter,
+  SorobanEffectFilter,
+  SubqlBlockHandler,
+  SubqlTransactionHandler,
+  SubqlOperationHandler,
+  SubqlEffectHandler,
 } from '@subql/types-soroban';
 import {plainToClass, Transform, Type} from 'class-transformer';
 import {IsArray, IsEnum, IsInt, IsOptional, IsString, IsObject, ValidateNested} from 'class-validator';
+import {Horizon, ServerApi} from 'stellar-sdk';
 import {SubqlSorobanProcessorOptions} from './types';
 
+export class BlockFilter implements SorobanBlockFilter {
+  @IsOptional()
+  @IsInt()
+  modulo?: number;
+  @IsOptional()
+  @IsString()
+  timestamp?: string;
+}
+
+export class TransactionFilter implements SorobanTransactionFilter {
+  @IsOptional()
+  @IsString()
+  account?: string;
+}
+
+export class OperationFilter implements SorobanOperationFilter {
+  @IsOptional()
+  type: Horizon.OperationResponseType;
+
+  @IsOptional()
+  @IsString()
+  source_account?: string;
```

Comment:
Would `from` be a better name?

Reply:
I used the same name as in the operation response.